angularjs-portal-home/src/main/webapp/my-app/layout/widget/controllers.js

```diff
@@ -104,12 +109,7 @@ define(['angular'], function (angular) {
     $scope.cycleUnits = function (){
 
       var userPreference = $scope.nextUnits;
-      if(userPreference === "" || userPreference === null || typeof userPreference === "undefined"){
-        userPreference = 'F';
-         $scope.initialPreference = true;
-      }else{
-         $scope.initialPreference = false;
-      }
+
 
       if(userPreference === 'F'  && !$scope.initialPreference){
```

Comment:
do we need the check of initial preference anymore?  Looks like it's always set to true now.